### ISSUE TITLE
--pe parameter question

### ISSUE THREAD

**Issue by Eric4Wu** [zstephens/neat-genreads#30]
Hi zstephens,

I am a new in simulation, I occasionally find your paper in plos one journal.

so I decide to try your simulator to generate simulated normal/tumor to evaluate CNV tools

In case of --pe parameter, can I interpret this to be "Mean insert size" ?

in our lab real data ,  the expected insert size is 500 bp 

so can I set this parameter to be "--pe 500 30"?

Many thanks!

**Comment by zstephens** [comment]
Yes, that would be the correct interpretation! --pe 500 30 would mean the fragment lengths would be sampled from a normal distribution with mean 500 and standard deviation 30.

Note that for simulation purposes the distributions are clipped at +- 6 standard deviations, so the possible values for --pe 500 30 would be [320,680].

Hope this helps!

**Comment by Eric4Wu** [comment]
Hi zstephens,

first question:

I have read your paper published in plos one, I don't see the Copy number variation frequency are 

included in genMutModel.py , does this mean I won't get CNV simulation in simulated reads even if 

there are CNVs record in VCF ? (>1000bp)

**Comment by tloka** [comment]
@Eric4Wu As far as I understand, CNVs are not simulated internally by neat. But you can use the `-v` parameter to include CNVs to your simulated data.
